libs/journey-bundles/transfer/src/lib/route.ts

```diff
@@ -1,11 +1,89 @@
 import { AuthGuard } from '@backbase/shared/feature/auth';
 import { SharedUserContextGuard } from '@backbase/shared/feature/user-context';
+import { Route } from '@angular/router';
+import { inject } from '@angular/core';
+import { JourneyCommunicationService } from '@backbase/shared/feature/communication';
+import { SHARED_JOURNEY_CONFIG } from '@backbase/shared/util/config';
+import { provideJourneyTracker } from '@backbase/foundation-ang/observability';
+import type { MakeTransferJourneyConfiguration } from '@backbase/transfer-journey';
 
-export const TRANSFER_ROUTE = {
+export const TRANSFER_ROUTE: Route = {
   path: 'transfer',
-  loadChildren: () =>
-    import('./transfer-journey-bundle.module').then(
-      (m) => m.TransferJourneyBundleModule
-    ),
+  loadChildren: async () => {
+    const [journeyModule, featureModule, dataAccessModule] = await Promise.all([
+      import('@backbase/transfer-journey'),
+      import('@backbase/transfer-journey/internal/feature'),
+      import('@backbase/transfer-journey/internal/data-access'),
+    ]);
+
+    return [
```

Comment:
This also doesn't look like the usual patterns for loading a journey - again we'd normally have a `transfer-journey-bundle` ts file that we lazy-load alone for loadChildren, and that file will export the routes as a default export, with the routes coming from the journey's entry function in the `@backbase/transfer-journey` dir.  We should never be importing directly from `internal` entry points of other packages.